spec/factories/involved_civilian.rb

```diff
@@ -11,7 +11,7 @@
     received_force_type { InvolvedCivilian::RECEIVED_FORCE_TYPES.sample_one_or_two_elements }
     received_force_location ['Head']
     injured false
-    custody_status 'In custody'
+    custody_status 'In custody (W&I section 5150)'
```

Comment:
This ends up rendering funny because the line is long:

<img width="779" alt="screen shot 2016-11-02 at 11 16 46 am" src="https://cloud.githubusercontent.com/assets/1421476/19941956/52313434-a0ef-11e6-92b8-40a77ee1ce6c.png">


You can fix this by changing the width of these divs -- see edit_civilian.html line 22
```
    <%= radio_button_grid(f, :custody_status, InvolvedCivilian::CUSTODY_STATUS_TYPES, 150) %>
```
You can change that 150 to 200 and it should fit, like this:

<img width="764" alt="screen shot 2016-11-02 at 11 27 32 am" src="https://cloud.githubusercontent.com/assets/1421476/19941970/5d4ebd32-a0ef-11e6-89a7-09d334bcf321.png">